api/v1beta1/machine_types.go

```diff
@@ -200,22 +200,70 @@ const (
 	// MachineHealthCheckSucceededV1Beta2Condition is true if MHC instances targeting this machine report the Machine
 	// is healthy according to the definition of healthy present in the spec of the MachineHealthCheck object.
 	MachineHealthCheckSucceededV1Beta2Condition = "HealthCheckSucceeded"
 
+	// MachineHealthCheckPassedV1Beta2Reason surfaces when a machine passes all the health checks defined by a MachineHealthCheck object.
+	MachineHealthCheckPassedV1Beta2Reason = "HealthCheckPassed"
+
+	// MachineHealthCheckUnhealthyNodeV1Beta2Reason surfaces when the node hosted on the machine do not pass the health checks
+	// defined by a MachineHealthCheck object.
+	MachineHealthCheckUnhealthyNodeV1Beta2Reason = "UnhealthyNode"
+
+	// MachineHealthCheckNodeStartupTimeoutV1Beta2Reason surfaces when the node hosted on the machine does not appear within
+	// he timeout defined by a MachineHealthCheck object.
+	MachineHealthCheckNodeStartupTimeoutV1Beta2Reason = "NodeStartupTimeout"
+
+	// MachineHealthCheckNodeDeletedV1Beta2Reason surfaces when a MachineHealthCheck detect that the node hosted on the
+	// machine has been deleted while the Machine is still running.
+	MachineHealthCheckNodeDeletedV1Beta2Reason = "NodeDeleted"
+
+	// MachineHealthCheckManuallyRemediatedV1Beta2Reason surfaces a MachineHealthCheck detects a machine manually remediated
+	// via the remediate-machine annotation.
+	MachineHealthCheckManuallyRemediatedV1Beta2Reason = "ManuallyRemediated"
+)
+
+// Machine's OwnerRemediated conditions and corresponding reasons that will be used in v1Beta2 API version.
+// Note: OwnerRemediated condition is created by the MachineHealthCheck controller, and then Set by the Machine's owner controller.
+const (
 	// MachineOwnerRemediatedV1Beta2Condition is only present if MHC instances targeting this machine
 	// determine that the controller owning this machine should perform remediation.
 	MachineOwnerRemediatedV1Beta2Condition = "OwnerRemediated"
+
+	// MachineOwnerRemediatedWaitingForOwnerV1Beta2Reason surfaces the machine is waiting for the owner controller
+	// to start remediation.
+	MachineOwnerRemediatedWaitingForOwnerV1Beta2Reason = "WaitingForOwner"
+)
+
+// Machine's ExternallyRemediated conditions and corresponding reasons that will be used in v1Beta2 API version.
+// Note: ExternallyRemediated condition is created by the MachineHealthCheck controller, and then Set by the external
+// remediation controller.
+const (
+	// MachineExternallyRemediatedV1Beta2Condition is only present if MHC instances targeting this machine
+	// determine that an external controller should perform remediation.
+	MachineExternallyRemediatedV1Beta2Condition = "ExternallyRemediated"
+
+	// MachineExternallyRemediatedWaitingForExternalRemediationV1Beta2Reason surfaces the machine is waiting for the
+	// external remediation controller to start remediation.
+	MachineExternallyRemediatedWaitingForExternalRemediationV1Beta2Reason = "WaitingForExternalRemediation"
```

Comment:
What about also using `WaitingForRemediation` (as I suggested above for OwnerRemediated).

I think the condition name already makes clear that we're talking about external remediation